app/components/grid.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import clsx from 'clsx'
+import type {ReactNode} from 'react'
+
+interface GridProps {
+  children: ReactNode
+  overflow?: boolean
+  className?: string
+}
+
+function Grid({children, className}: GridProps) {
+  return (
+    <div
+      className={clsx(
+        'gap-[18px] mx-[10vw] md:gap-[17px] lg:gap-[23px] grid grid-cols-4 md:grid-cols-8 lg:grid-cols-12',
+        className,
+      )}
+    >
+      {children}
+    </div>
+  )
+}
+
+/**
+ * Use for development only! It renders the grid columns and gaps as page overlay
```

Comment:
Really useful while creating new components/sections. Just add this one to your markup, and you'll have a nice visual column grid. 

```jsx
<>
  <GridLines />
  <Grid>
    <div ...
  </Grid>
<>
```

Reply:
Handy!